README.md

```diff
@@ -1 +1,32 @@
-# riscv
+﻿RV32G Processor model
+===================================================================
+
+This is the functional model of RISC-V instruction set architecture. It contains all the instructions necessary for running mibench programs. It has following types of instruction encoding:
```

Comment:
In general, when writing text files, never write more than 80 chars per line, so it nicely fits any terminal screen. For code, clang-format automatically does that for you, but for text files, you should be manually fixing this (or use an editor that breaks lines automatically).